msal/src/main/java/com/microsoft/identity/client/AuthenticationResult.java

```diff
@@ -51,6 +52,19 @@ public AuthenticationResult(final ICacheRecord cacheRecord) {
         mAccount = AccountAdapter.adapt(cacheRecord.getAccount());
     }
 
+    public AuthenticationResult(AccessTokenRecord accessToken,
+                                String tenantId,
```

Comment:
If this `AccessTokenRecord` is fully-formed, the `realm` property of this object should match the `tenantId` - can we lose this parameter? Unclear what it would mean if these values were different